radixdlt-java-common/src/main/java/com/radixdlt/errors/RadixErrors.java

```diff
@@ -62,38 +63,97 @@
  * permissions under this License.
  */
 
-package com.radixdlt.identifiers;
+package com.radixdlt.errors;
 
 import com.radixdlt.utils.functional.Failure;
 
-public enum CommonErrors implements Failure {
-	UNKNOWN_ERROR("Unknown error of type {0} with message {1}"),
+/**
+ * Full list of internal error codes.
```

Comment:
I don't think this should be a full list of internal codes, only codes which will be passed to api layer. Internal errors should be managed via different exception classes.

Reply:
I believe that we should not use exceptions at all, as they're making code very hard to follow and reason about. But I agree that we should have codes separated for internal errors and errors which may appear at API level.